src/main/kotlin/com/cosmotech/connector/adt/utils/JsonUtil.kt

```diff
@@ -52,13 +41,14 @@ class JsonUtil {
          * @return a Pair(Boolean,String) containing minimal extension information
          */
         @JvmStatic
-        fun isExtension(jsonModel: JsonObject):Pair<Boolean,String> {
-            return if( jsonModel.containsKey(DTDL_EXTENDS_KEY) )
-                Pair(true,jsonModel[DTDL_EXTENDS_KEY].toString())
+        fun isExtension(jsonModel: JsonObject):Pair<Boolean,List<String>> {
+            val result = jsonModel.array<String>(DTDL_EXTENDS_KEY)
+            return if(result != null)
+                Pair(true, result)
             else
-                Pair(false,"")
+                Pair(false, listOf<String>())
```

Comment:
No need to repeat `String` here, wich will be inferred. So just `listOf()` should be fine, I think.

Reply:
done